src/pages/Committees.js

```diff
@@ -587,4 +641,204 @@ function SportsCommittee() {
     )
 }
 
+function Styrelsen() {
+    return (
+        <p>
+            The chapter board (Swedish: Styrelsen) is the organ of the Media Technology chapter that oversees the chapter's activities.
+            All committees' chairperson report to the board, which in turn discuss and makes decisions at the highest level in the chapter.
+            Joining the board is an opportunity to influence the chapter. The board is also responsible for business issues and study monitoring.
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: styrelsen@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function MediaElectionCommittee () {
+    return (
+        <p>
+            The Election Committee (Swedish: Valberedningen) prepares and manages the elections for certain positions in the Media chatper.
+            The committee receives nominations and candidacies, asses candidates with an interview, and make a review of the candidates for
+            the next chapter meeting. The Election Committee works to keep elections democratic and ensure all candidates have an equal treatment
+            and process.
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: val@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function MediaStudyBoard() {
+    return (
+        <p>
+            The Study Board (Swedish: Studienämnden) verifies that every student has a well-functioning education and safe
+            study environment. This applies at both programme-level and course-level. If you have any problem with courses and
+            programs, or have questions about them, you can contact the Study Board! Joining the committee lets you help everyone
+            have a better education and you will have a better understanding of education at KTH.
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: sno@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function CommunicationCommittee () {
```

Comment:
Short: KomN
They also create merch for the chapter and host a "christmas calendar"? (swedish: julkalender; a mini series consisting of 24 short videos, released one each day leading up to christmas eve) each christmas. I'm also not sure if they "take care of communications within the chapter" as our internal communications are basically just slack and discord.

Reply:
I based myself on their description on Media's website, and either it changed (the website got updated anyway) or I just misread.
Right now the website says "KomN ansvarar även för intern kommunikation inom sektionen genom att till exempel hålla i roliga tävlingar, skapa aftermovies åt olika event på sektionen, spela in Medias helt egna julkalender och lansera merch åt sektionen.". So it seems its closer to team buildings. Is that the case in your experience?

Reply:
That's correct, maybe not as in physical teambuildings, but they do fun PR stuff that helps bring the chapter together. A bit of a stretch to call that "internal communication" in my opinion